.eslintrc.yml

```diff
@@ -9,3 +13,17 @@ rules:
     - error
     - commonjs: true
       ignore: [ ^aws-sdk$, ^oracledb$, ^request-promise-native$ ]
+  jsdoc/require-param:
+    - warn
+    - exemptedBy: [ type ]
+  jsdoc/require-returns:
+    - warn
+    - exemptedBy: [ type ]
+  jsdoc/no-undefined-types:
+    - warn
+      # Express types
```

Comment:
An extra space before `#`?

Reply:
Oh, I was intending the comment to line up with `definedTypes`. Should this be moved somewhere else?